api/test/integration/fetchRecord.test.js

```diff
@@ -0,0 +1,283 @@
+        housingRegister: [],
+        name: [{ first: 'Arlyn', last: 'Wilce', title: 'Ms' }],
+        postcode: [],
+        systemIds: { uhw: ['4186867'] }
+      }
+    });
+  });
+
+  it('returns info for customer with Jigsaw record', async () => {
+    const response = await doSearchRequest(`${BASE_URL}/customers/126/record`);
+    expect(response).toStrictEqual({
+      customer: {
+        address: [{ address: ['Hackney London W3 43no'], source: ['JIGSAW'] }],
+        dob: ['1991-02-13 12:00:00'],
+        email: ['james@hotmail.com'],
+        housingNeeds: {
+          currentPlacement: {
+            address: 'Room 1 hallway drive ',
+            duty: 'Section 192',
+            rentCostCustomer: 0,
+            startDate: '2019-04-05 12:00:00',
+            tenancyId: 64444,
+            type: 'Accommodation secured by the Local Authority'
+          },
+          jigsawCaseId: '54321',
+          status: 'Open'
+        },
+        housingRegister: [],
+        nhsNumber: '',
+        nino: ['ABC12345D'],
+        phone: ['07666666666'],
+        systemIds: { jigsaw: ['12345'] },
+        team: {}
+      }
+    });
+  });
+
+  it('returns info for customer with Academy-Benefits record', async () => {
+    const response = await doSearchRequest(`${BASE_URL}/customers/127/record`);
+    expect(response).toStrictEqual({
+      customer: {
+        address: [
+          {
+            address: [
+              '8017 Garrison Point',
+              '2 Lake View Crossing',
+              'London',
+              'S3 1EV'
+            ],
+            source: ['ACADEMY-Benefits']
+          }
+        ],
+        benefits: {
+          income: [
+            {
+              amount: 2.03,
+              description: 'Future-proofed motivating workforce',
+              frequency: 1,
+              period: 'Weekly'
+            },
+            {
+              amount: 2.03,
+              description: 'Virtual encompassing internet solution',
+              frequency: 1,
+              period: 'Weekly'
+            },
+            {
+              amount: 2.03,
+              description: 'Multi-lateral tertiary extranet',
+              frequency: 1,
+              period: 'Weekly'
+            },
+            {
+              amount: 2.03,
+              description: 'Advanced clear-thinking algorithm',
+              frequency: 1,
+              period: 'Weekly'
+            }
+          ],
+          live: false
+        },
+        dob: ['1981-02-08 12:00:00'],
+        housingNeeds: {},
+        housingRegister: [],
+        name: [{ first: 'Flor', last: 'Beden', title: 'Mr' }],
+        nino: ['CD877342Z'],
+        postcode: ['S3 1EV'],
+        systemIds: { academyBenefits: ['60605913'] }
+      }
+    });
+  });
+
+  it('returns info for customer with Academy-CouncilTax record', async () => {
+    const response = await doSearchRequest(`${BASE_URL}/customers/128/record`);
+    expect(response).toStrictEqual({
```

Comment:
Suggestion: Some people don't like them as writing out your expectations in the test is more explicit, but a possible option would be to use [snapshot tests](https://jestjs.io/docs/en/snapshot-testing) for these.